### DISCUSSION TITLE
Can I create a role in Ranger to manage the permissions of  a group of Doris users ?

### DISCUSSION THREAD

**Discussion by cutebear1228** [apache/doris#45464] [A - General / Q&A]
Hi all , 
I would like to know if Doris permisssion can be managed by role. I create a role (Tier1_Role) in Apache Ranger and assign a user to the specific role. However , the access policy did not apply to assigned role but worked when I assigned it to the specific user . 
I read the document but it mentioned only the user account . Does anyone know the settings if I want to use a role to manage the Doris permissions? Any help will be appreciated. Thanks.

Reference : 
https://doris.apache.org/docs/admin-manual/auth/ranger#configuring-permissions 

![image](https://github.com/user-attachments/assets/03b3bdb0-d25b-4a29-b147-b17831f750e4)

**Comment by gagan211** [comment]
@cutebear1228 i would like to ask you a few things if you can ping me back .
